spacy/tests/vocab_vectors/test_lexeme.py

```diff
@@ -1,7 +1,62 @@
-import pytest
 import numpy
+import pytest
 from spacy.attrs import IS_ALPHA, IS_DIGIT
+from spacy.lookups import Lookups
+from spacy.tokens import Doc
 from spacy.util import OOV_RANK
+from spacy.vocab import Vocab
+
+
+@pytest.mark.issue(361)
+@pytest.mark.parametrize("text1,text2", [("cat", "dog")])
+def test_issue361(en_vocab, text1, text2):
+    """Test Issue #361: Equality of lexemes"""
+    assert en_vocab[text1] == en_vocab[text1]
+    assert en_vocab[text1] != en_vocab[text2]
+
+
+@pytest.mark.issue(595)
+@pytest.mark.skip(reason="Old vocab-based lemmatization")
+def test_issue595():
+    """Test lemmatization of base forms"""
+    words = ["Do", "n't", "feed", "the", "dog"]
+    lookups = Lookups()
+    lookups.add_table("lemma_rules", {"verb": [["ed", "e"]]})
+    lookups.add_table("lemma_index", {"verb": {}})
+    lookups.add_table("lemma_exc", {"verb": {}})
+    vocab = Vocab()
+    doc = Doc(vocab, words=words)
+    doc[2].tag_ = "VB"
+    assert doc[2].text == "feed"
+    assert doc[2].lemma_ == "feed"
+
+
```

Comment:
```suggestion
```

This test isn't going to work in v3 at all and should just be removed.

Reply:
Ah that's a good point, tests that are old and currently skipped because they became redundant, can mostly just be removed.

Reply:
Resolved in https://github.com/explosion/spaCy/pull/9655/commits/65f66d4e0384e1d15d60384f2cdc13d35d2991a6